dmutils/email/dm_mandrill.py

```diff
@@ -6,6 +6,7 @@
 from mandrill import Mandrill
 from dmutils.email.exceptions import EmailError
 from dmutils.email.helpers import hash_string
+from dmutils.timing import logged_duration_for_external_request as log_external_request
```

Comment:
If we don't like the name of this thing, then maybe now is the time to change it, cc @risicle

Reply:
I called it `logged_duration` so that the line reads `with logged_duration...` i.e. "do the following with its duration being logged...

What other people want to call other things I don't know. I can see that here it's possibly ok to alias it to a shorter-but-less-clear name for brevity, but newcomers to `timing.py` would still see the function with its proper name for clear signposting.